L/Libtask/build_tarballs.jl

```diff
@@ -5,7 +5,7 @@ using BinaryBuilder, Pkg
 julia_version = v"1.6.0"
```

Comment:
Just to be kinda safer, maybe you should build against libjulia 1.3?

Reply:
Update.